src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+import core.basesyntax.model.abstraction.Color;
+import core.basesyntax.model.abstraction.Figure;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle() {
+    }
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.ceil(Math.PI * radius * radius);
+    }
+
+    @Override
+    public String toString() {
```

Comment:
In order to do this, you must create another interface with a draw method that outputs data about the figure.